src/bb-functions-new.ts

```diff
@@ -0,0 +1,21 @@
+import program from 'commander';
+import path from 'path';
+import { newFunctionDefinition } from './functions/functionDefinitions';
+
+/* process arguments */
+program
+  .usage('[function-name]')
+  .name('bb functions new')
+  .parse(process.argv);
+
+const {
+  args: [inputFunctionName],
+} = program;
+
+(async (): Promise<void> => {
```

Comment:
there is no await in use, so I think we can unrwap this

Reply:
Good call. Forgot to clean this up after changing another function to sync.